covasim/parameters.py

```diff
@@ -146,9 +146,9 @@ def get_prognoses(by_age=True):
             death_probs  = np.array([0.00002, 0.00006, 0.00030, 0.00080, 0.00150, 0.00600, 0.02200, 0.05100, 0.09300]), # Overall probability of dying (https://www.imperial.ac.uk/media/imperial-college/medicine/sph/ide/gida-fellowships/Imperial-College-COVID19-NPI-modelling-16-03-2020.pdf)
         )
 
-    prognoses['death_probs']  /= prognoses['crit_probs']   # Conditional probability of dying, given severe symptoms
-    prognoses['crit_probs']   /= prognoses['severe_probs'] # Conditional probability of symptoms becoming critical, given severe
-    prognoses['severe_probs'] /= prognoses['symp_probs']   # Conditional probability of symptoms becoming severe, given symptomatic
+    prognoses['death_probs']  = prognoses['death_probs'] / prognoses['crit_probs']   # Conditional probability of dying, given critical symptoms
+    prognoses['crit_probs']   = prognoses['crit_probs'] / prognoses['severe_probs'] # Conditional probability of symptoms becoming critical, given severe
+    prognoses['severe_probs'] = prognoses['severe_probs'] / prognoses['symp_probs']   # Conditional probability of symptoms becoming severe, given symptomatic
```

Comment:
Hmm, would you be able to give an example? I'm not able to reproduce this

Reply:
Example code:
```
import covasim as cv
sim = cv.Sim(prog_by_age=False)
sim.run()
sim['prognoses']['death_probs'][0] # 0.25, as we would expect
```

Reply:
Hm. I just tried and was also unable to reproduce it. Spoooky. I was definitely getting an error which StackOverflow helped me fix this way.. but now I can't repro it either. Feel free to ignore this part of PR, Can you just get the test and ignore changes to this file?